### PR TITLE
chore: Introduce Changesets

### PR DESCRIPTION
**Pull request by chaance**
Closes: #

- [ ] Docs
- [ ] Tests

Testing Strategy:

### CODE REVIEW

**Review by kentcdodds — COMMENTED** [review]
Looking good! Could we get docs added for the workflow on how to do releasy things?

File: .changeset/config.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "remix",
+      "remix-cloudflare",
+      "remix-deno",
+      "remix-dev",
+      "remix-node",
+      "remix-react",
+      "remix-serve",
+      "remix-server-runtime"
+    ]
+  ],
```
[the remark this answers is not in the document]
Comment (chaance) [reply .changeset/config.json:16]: The platform adapters are not included in `fixed`, only core and runtime are. We can split out the runtime packages as well if that's what we want to do. Internally we've separated those from the adapters so I wasn't sure what we wanted to do with them.

File: package.json [hunk 2]
```diff
@@ -31,15 +31,18 @@
...
     "playground:new": "node ./scripts/playground/new.js",
     "publish": "node ./scripts/publish.js",
-    "publish:private": "node ./scripts/publish-private.js",
-    "release": "node ./scripts/release.js",
     "test": "run-p \"test:*\"",
     "pretest:integration": "rollup -c",
     "test:integration": "playwright test --config ./integration/playwright.config.ts",
     "posttest:integration": "rimraf ./.tmp/integration",
     "test:primary": "jest",
-    "version": "node ./scripts/version.js",
```
[the remark this answers is not in the document]
Comment (chaance) [reply package.json:41]: I want to keep them for now, they might be useful for cutting manual releases. Though I do think we are probably safe to finally get rid of `publish-private.js` if everyone agrees.

File: .changeset/config.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (kentcdodds) [reply .changeset/config.json:16]: Gotcha. Isn't cloudflare a platform adapter?

[the remark this answers is not in the document]
Comment (chaance) [reply .changeset/config.json:16]: Technically a runtime! It powers `cloudflare-pages` and `cloudflare-workers` which are the adapters.

**Pull request closed without merging**